itsUk45/Property-Pro-Lite-
# #167270354 feature develop api test mocha

#### What does this PR do?
- adds  test cases to api routes

#### Description of Task to be completed
- Before a user perform any operation, a test case is executed. 

#### How should this be manually tested?
- clone the repo and switch to the branch ft-develop-api-test-moocha
- run nodemon app to start the server
- run npm run coveralls to see the test output

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[#2354662](http://pivotaltracker.com)

#### Screenshots (if appropriate)
 -N/A
